doc/devel/coding_guide.rst

```diff
@@ -126,37 +196,68 @@ PR Review guidelines
   with the contributor first.
 
 
+.. _branches_and_backports:
+
 Branches and Backports
 ======================
 
-
+Current branches
+----------------
 The current active branches are
 
 *master*
-  This will be Matplotlib 3.0.  Supports Python 3.5+.
+  The current development version. Future minor releases (*v3.N.0*) will be
+  branched from this. Supports Python 3.6+.
+
+*v3.N.x*
+  Maintenance branch for Matplotlib 3.N. Future patch releases will be
+  branched from this.  Supports Python 3.6+
```

Comment:
Not true for `v3.0.x` branch. Also missing a final period.

Reply:
While still there, `v3.0.x` is not really active. I don't think well push anything to that branch anymore. Therefore I wouldn't bother with a distinction.

The other question is if stating the supported Python version is reasonable here. I've just taken that from the existing descriptions.

Reply:
It does seem a bit redundant, but since PRs always target `master`, it's useful to know this before the backport happens.